examples/deck.gl/3d-tiles/package.json

```diff
@@ -35,6 +35,7 @@
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.5",
     "babel-plugin-inline-import": "^3.0.0",
+    "babel-plugin-version-inline": "^1.0.0",
```

Comment:
Do we need both of those plugins? What is the difference?

Reply:
Ignore that comment. I suppose this plugin is needed by start-local and ideally it should be injected by the webpack-local config?